src/pdmat.jl

```diff
@@ -26,6 +26,7 @@ Base.convert(::Type{AbstractArray{T}}, a::PDMat) where {T<:Real} = convert(PDMat
 dim(a::PDMat) = a.dim
 Base.Matrix(a::PDMat) = copy(a.mat)
 LinearAlgebra.diag(a::PDMat) = diag(a.mat)
+LinearAlgebra.cholesky(a::PDMat) = copy(a.chol)
```

Comment:
I'm not sure this one should copy.

Reply:
I'm more confident that it should be a copy if we're going to support `PDiagMat` which wouldn't be a strict accessor

Reply:
We usually do the minimal amount of copying required and allow the aliasing behavior to be different for different types so I still don't think this should copy.